src/main/java/org/fundacionjala/pivotal/config/PivotalEnvironment.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.pivotal.config;
+
+import org.fundacionjala.core.config.Environment;
+
+public class PivotalEnvironment extends Environment {
+    /**
+     * Initializes an instance of properties files.
+     *
+     * @param propertiesPath
+     */
+    public PivotalEnvironment(final String propertiesPath) {
+        super(propertiesPath);
+    }
+
+    /**
+     * get the token from the file.properties.
+     *
+     * @return token value.
```

Comment:
consider removing the * from the lines 8 and 17

Reply:
I think it is ok to separate the description from the @params or @return.